### PR TITLE
added multiple azure managed identities documentation

### PR DESCRIPTION
**Pull request by tomasfreund**
Signed-off-by: Tomáš Freund <tomas.freund@datamole.cz>

Documentation for pull request https://github.com/jetstack/cert-manager/pull/4332

### CODE REVIEW

**Review by jakexks — CHANGES_REQUESTED** [review]
Hi @tomasfreund 

If this is documentation for a feature that will be in 1.6, please could you make this PR against the release-next branch? Anything merged into master is live on the website, which could be confusing for users seeing documentation for a feature that does not yet exist.

File: content/en/docs/configuration/acme/dns01/azuredns.md [hunk 1]
```diff
@@ -180,7 +179,16 @@ resource "azurerm_role_assignment" "dns_contributor" {
 }
 ```
 
-Then when creating the cert-manager issuer we only need to specify the `hostedZoneName`, `resourceGroupName` and `subscriptionID` fields for the DNS Zone. Example below:
+Then when creating the cert-manager issuer we only need to specify the `hostedZoneName`, `resourceGroupName` and `subscriptionID` fields for the DNS Zone.
```
Comment (jakexks) [inline content/en/docs/configuration/acme/dns01/azuredns.md:182]:
```suggestion
Then when creating the cert-manager issuer we need to specify the `hostedZoneName`, `resourceGroupName` and `subscriptionID` fields for the DNS Zone.
```

**Comment by tomasfreund** [comment]
Hi, is there anything more I need to do to get this and https://github.com/jetstack/cert-manager/pull/4332 merged?

**Review by wallrj — COMMENTED** [review]
Thanks @tomasfreund 

I quoted a couple of words to satisfy the spell-checker and cleaned up some trailing whitespace.

Thanks again for all your work on this.

/lgtm
/approve

**Pull request merged**